### ISSUE TITLE
Ability to run collections from the command line

### ISSUE THREAD

**Issue by Fillps** [rangav/thunder-client-support#759]
With the extended support to run collections, it would be nice to run from a command line environment. Not sure if there is a way to do that yet, but it would be helpful to test the API in the CI/CD pipeline.

**Describe the solution you'd like**
Having the ability to run a command like:
```
$ thunder-tests --path "my-thunder-tests-folder" --collection "My collection"
Running collection "My collection"
Requests                             Status        Time       Tests
My Request 1                         200           20ms       1/1
My Request 2                         200           20ms       1/1

Tests ran successfully.
```
It will be also nice to have the output of the test in a file with a standard test format so the CI/CD pipeline can parse them. 

I know this is a big ask, but want to have your thoughts about this feature.

**Comment by alexbenfica** [comment]
This is an excellent idea. 

Given my current development setup using Thunder Client, we would be able to re-use all local tests as a comprehensive test to run against our backends. Also, it makes it a lot easier if the visual interface can be used while creating tests that will then run in the CI/CD environment.

**Comment by rangav** [comment]
@Fillps thanks for the feedback, this feature is already in priority list. will implement soon.

duplicate of #98

**Comment by rangav** [comment]
This feature is now implemented and published to marketplace, please update to `v2.4.0`

### Thunder Client CLI
- https://github.com/rangav/thunder-client-support/releases/tag/v2.4.0

### Documentation
- For documentation read our blog post on [medium.com](https://rangav.medium.com/d91eb5c71d8e)

### Product Hunt
- We are launching today on [Product Hunt](https://www.producthunt.com/posts/thunder-client-cli), Please **upvote** and let us know any feedback.

**Comment by rangav** [comment]
Hi, Now you can run **Multiple or All Collections** from CLI in `v1.1.0`

- Run **All** Collections using `tc --col all`
- Run **Multiple** Collections `tc --col colIdOrName1,colIdOrName2`
- Update CLI to version - **v1.1.0**

Release Notes:
https://github.com/rangav/thunder-client-support/releases/tag/v2.5.0

Please let me know your feedback.
